stable/rabbitmq/templates/_helpers.tpl

```diff
@@ -29,4 +29,19 @@ Create chart name and version as used by the chart label.
 */}}
 {{- define "rabbitmq.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Return the appropriate apiVersion for APIs.
+*/}}
+{{- define "rabbitmq.workloadApiVersion" -}}
```

Comment:
I would add a small comment in each branch to indicate the k8s version that it is applicable.